Abot2/Core/AngleSharpHyperLinkParser.cs

```diff
@@ -57,6 +58,32 @@ protected override string GetBaseHrefValue(CrawledPage crawledPage)
             return baseTagValue.Value.Trim();
         }
 
+        protected override string GetMetaRedirectUrl(CrawledPage crawledPage)
```

Comment:
Can we add some unit tests that exercise this code please? They can be found in HyperlinkParserTest.cs file. Please be sure to test variability of that value (success and failure cases). Two classes inherit from this test file (AngleSharpHyperLinkParser.cs and HapHyperlinkParser.cs) that must both pass the tests in their base class.